backend/api/build_kfp_server_api_python_package.sh

```diff
@@ -47,15 +50,19 @@ DIR=$(mktemp -d)
 
 swagger_file=$(mktemp)
 
+# Documentation for info object can be found at https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#infoObject
 echo "Merging all Swagger API definitions to $swagger_file."
 jq -s '
     reduce .[] as $item ({}; . * $item) |
     .info.title = "KF Pipelines API" |
-    .info.description = "Generated python client for the KF Pipelines server API"
+    .info.description = "Generated python client for the KF Pipelines server API" |
+    .info.version = "'$VERSION'" |
+    .info.contact = { "name": "google", "email": "kubeflow-pipelines@google.com", "url": "https://www.google.com" } |
+    .info.license = { "name": "Apache 2.0", "url": "https://raw.githubusercontent.com/kubeflow/pipelines/master/LICENSE" }
 ' ./swagger/{run,job,pipeline,experiment,pipeline.upload}.swagger.json > "$swagger_file"
 
 echo "Generating python code from swagger json in $DIR."
-java -jar "$codegen_file" generate -l python -i "$swagger_file" -o "$DIR" -c <(echo '{
+java -jar "$codegen_file" generate -l python -i "$swagger_file" -t "./python_client_template" -o "$DIR" -c <(echo '{
```

Comment:
a better folder name to let others aware it's auto-generated and no manual edit?

I don't know why we must submit the auto-gen files as normally bazel based project don't do it. The PR title says license but it seems the .mustache is not related with license. Any info?

Reply:
@rmgogogo These are not auto generated, they are copied from https://github.com/swagger-api/swagger-codegen/blob/v2.4.7/modules/swagger-codegen/src/main/resources/python, because the original setup.mustache file doesn't specify author and license fields, so I patched the setup file with KFP specific values (if you look at commit history it's very clear).